apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -48,56 +84,194 @@ class NewReward extends React.Component {
     dataToSend.disbursementCycle = disbursementCycles[this.state.disbursementCycle]
     dataToSend.disbursementDelay = disbursementDates[this.state.disbursementDate]
     dataToSend.isMerit = !dataToSend.rewardType ? true : false
-    dataToSend.referenceAsset = this.props.balances[this.state.referenceAsset+1].address // account for no ETH in reference asset dropdown
+    dataToSend.referenceAsset = getTokenProp('address', this.props, this.state,'isVerified')
     this.props.onNewReward(dataToSend)
   }
 
   canSubmit = () =>
     !(
       this.state.amount > 0 &&
       this.state.description !== '' &&
-      this.state.dateEnd > this.state.dateStart
+      this.state.dateEnd > this.state.dateStart &&
+      (
+        this.state.referenceAsset > 1 ||
+        this.state.customToken.address
+      ) &&
+      !this.errorPrompt()
     )
 
+  startBeforeTokenCreation = () => (getTokenProp('startBlock',this.props,this.state)) > this.state.startBlock
+  disbursementOverflow = () => (this.state.quarterEndDates ? this.state.quarterEndDates.length > 41 : false)
+  lowVaultBalance = () => this.props.balances[this.state.amountCurrency].amount / Math.pow(10,this.props.balances[this.state.amountCurrency].decimals) < this.state.amount
+  dividendPeriodTooShort = () => (this.state.rewardType > 0 && this.state.occurances === 0)
+  errorPrompt = () => (this.showSummary() && (this.startBeforeTokenCreation() || this.disbursementOverflow() || this.lowVaultBalance() || this.dividendPeriodTooShort()))
+
+  showSummary = () => (this.state.referenceAsset > 1 || this.state.customToken.symbol)
+
+  getItems() {
+    if (!this.props.refTokens) {
+      return ['Tokens Loading...']
+    }
+    return [ 'Select a token', 'Other…', ...this.getTokenItems() ]
+  }
+
+  getTokenItems() {
+    return this.props.refTokens
+      .filter(token => token.startBlock ? true : false)
+      .map(({ address, name, symbol, verified }) => (
+        <TokenSelectorInstance
+          address={address}
+          name={name}
+          showIcon={verified}
+          symbol={symbol}
+        />
+      ))
+  }
+
+  handleCustomTokenChange = event => {
+    const { value } = event.target
+    const { network } = this.props
+
+    // Use the verified token address if provided a symbol and it matches
+    // The symbols in the verified map are all capitalized
+    const resolvedAddress =
+      !isAddress(value) && network.type === 'main'
+        ? ETHER_TOKEN_VERIFIED_BY_SYMBOL.get(value.toUpperCase()) || ''
```

Comment:
I know it's sort of an edge case but there are duplicates in this list and there isn't really any indication in the UI that this:
1. Only works on mainnet
2. Is pulling symbols off of the verified token list
I think maybe just some sort of warning messages around this functionality would be good.

Reply:
I've updated the UX so it only prompts for an address on any network that isn't mainnet. I've also made the prompts contextually adaptive